### ISSUE TITLE
Strange behvaior in the network. Am I doing something wrong or is there a bug?

### ISSUE THREAD

**Issue by henkaa** [AppliedEnergistics/Applied-Energistics-2#5331]
**Describe the bug**
The network was function just fine but suddenly it started to behave in stange manners. Now everytime I start the game, the networks seems to work until I try to craft anything, then it stops working and I cannot do anything in the network.

**To Reproduce**
1, Start the game.
2. Notice that you can input output/input items without problems.
3. Try to craft something. The craft does not appear in the crafting terminal.
4. Try to output an item from the network, you cannot, it seems frozen.
5. Go to the drive and take out one disk then install it again.
6. Try to output an item from the network, you can get one stack, but all rest of the item "disaper"
7. Input the item again. it is inputable, but does not show in the network.
8. Close the crafting terminal and open it again. No you see no items at all in the network.
9. Go to the drive and take out one disk then install it again.
10. Open the terminal again and the items are back. Now you are back at step 6 basicly again.

**Expected behavior**
That the network works as intended.

**Additional context**
The complete network is fitted within 6 chunks (3*2) so it should be loaded atleast when I am around (which is most of the time)
All key components of the network are in the same chunk (disk drive, controllers, security terminal, dence energy cells, energy acceptor etc)
Everything seems to be online

I have tried with and without a Kibe chunk loader. Back when it worked, this was placed in the base.
I have tried to reduce surrounding "machines" that could produce lag.
I have tried to reduce the network, with no change in behavior.
I have a chest connected to an import bus. When the network still works (after start of game) items import to the network. As soon as the problem occur, items are no longer imported.

**Environment**
Tried with both Curseforge and MultiMC as starter.
Fabric loader 0.11.3
Modpack: All Of Fabric 3 v.2.8.5
AE2 mod v: 8.3.0-alpha3

**Network**
1 Energy acceptor. (powered by 1 Tech Reborn MFSU, powerd by 16 Tech Reborn Ultimate Solar Panels)
2 Controllers
2 Dence Energy Cells
1 ME Drive housing 3 64K item cells
1 Security Terminal
1 p2p tunnel that is linked
1 "multiblock" to autocraft 4 inscribers (follwed a guide online9
1 "multiblock" with 8 interfaces and 8 molecular assemblers
5 interaces connected to Industrial revolution machines
12 interfaces connected to Tech Reborn machines
3 import busses to get items back from the machines
1 Crafting terminal
1 Pattern Terminal
1 Interface Terminal
1 CPU consisting of
- 1 Co processor
- 1 64k storage
- 1 16k storage
- 1 4k storage

**Comment by henkaa** [comment]
Interesting note here. I removed a Conjuring setup just to try.. and it did not work while plying.. but on a restart of the game my ME network feels better, atm at least.. I will investigate further.

**Comment by henkaa** [comment]
The error reappeard so Conjuring was probably not the only thing, or there is another hidden problem.

**Comment by CygnusiaX1** [comment]
You stated you're using Tech Reborn machines, right? Are you, by chance, using the Recycler or Scrap-inator  directly connected to your network?

I am actually on my second world, the first of which I abandoned due to my network acting like this. The second time this problem started was after I connected those 2 machines to my network. I ended up keeping the machines in my inventory, pulling all scrap and scrapboxes out of the network, unplugging the network from power and letting the batteries drain dry (just in case). I also completely logged out and restarted the game (again, just in case.) Once my network was powered up again, the network seemed to be acting normally.

I ended up making a mini-network with those two machines, completely outside of the other network. I manually pull anything it makes and put them into the main network. I've avoided any "contamination" this way.

I know this is a bit TL;DR, but I hope this helps...
